app/src/main/java/com/github/se/cyrcle/model/parking/TestInstancesParking.kt

```diff
@@ -4,9 +4,24 @@ import com.mapbox.geojson.Point
 
 object TestInstancesParking {
 
-  val referencePoint = Point.fromLngLat(6.9, 46.69)
+  val referencePoint = Point.fromLngLat(6.55, 46.65)
   val EPFLCenter = Point.fromLngLat(6.566397, 46.518467)
 
+  val parkingEnd2End =
```

Comment:
I had to change the location of the  test Instance otherwise it wouldn't load on the list Screen as my prevent the List screen from loading the whole map.